terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/i18n/AbstractI18nCodeListTest.java

```diff
@@ -46,6 +48,7 @@ protected Map<String, String> obtainMap(Locale locale) {
             }
         };
 
+        originalLocale = Locale.getDefault();
         Locale.setDefault(Locale.ENGLISH);
         assertThat(impl.asMap(), hasEntry("language", Locale.ENGLISH
                 .getLanguage()));
```

Comment:
- [ ] Is there a reason why this is the only place in the test method where it is evicted?
If there is no particular reason, then it should be reverted before and after the test just like any other test.

Reply:
389c15f
For no particular reason, we modified the test to be undone before and after the test, just like any other test.

Reply:
Well... ``Locale.setDefault(Locale.ENGLISH);`` can be implemented in ``testAsMap``.

Reply:
09d3214
Corrected according to comments.